bin/local-env/install-wordpress.sh

```diff
@@ -111,6 +111,10 @@ container chmod 767 /var/www/html/favicon.ico
 echo -e $(status_message "Activating Web Stories plugin...")
 wp plugin activate web-stories --quiet
 
+# Install & activate testing plugins.
+echo -e $(status_message "Installing and activating RTL Tester plugin...")
+wp plugin install rtl-tester --activate --force --quiet
```

Comment:
FYI, the RTL Tester is already installed on the test environment. This is just for the local one.